corehq/apps/reports/standard/monitoring.py

```diff
@@ -1737,12 +1738,12 @@ def avg_datespan(self):
             avg_datespan.startdate = datetime.datetime(1900, 1, 1)
         return avg_datespan
 
-    def _report_data(self):
+    def _report_data(self, users_to_iterate, user_ids):
```

Comment:
It seems as if user_ids is always [u.user_id for u in users_to_iterate]. Is there a reason you make the caller pass it as a separate parameter, instead of doing this inside the method?

Reply:
Here it gets created from the generator: https://github.com/dimagi/commcare-hq/pull/23734/files#diff-30c4e1ef0d3ceb995aab05246948b579R1813
And when it's called here: https://github.com/dimagi/commcare-hq/pull/23734/files#diff-30c4e1ef0d3ceb995aab05246948b579R1823
it gets it from here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/standard/monitoring.py#L1464